app/test_main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+from unittest import mock
+import pytest as pytest
```

Comment:
`import pytest as pytest` - interesting, what is the purpose?)

Also, different groups of imports you should split with a blank line